### PR TITLE
webapi : Unequip item to upgrade

### PR DESCRIPTION
**Pull request by namidaka**

### CODE REVIEW

File: src/Application/Items/Commands/UpgradeUserItemCommand.cs [hunk 1]
```diff
@@ -89,6 +89,7 @@ public async Task<Result<UserItemViewModel>> Handle(UpgradeUserItemCommand req,
                 IsBroken = false,
             };
 
+            _db.EquippedItems.RemoveRange(userItemToUpgrade.EquippedItems);
```
Comment (verdie-g) [inline src/Application/Items/Commands/UpgradeUserItemCommand.cs:92]: The EquippedItems are not loaded. Also could you update the tests.

**Review by verdie-g — CHANGES_REQUESTED** [review]

File: src/Application/Items/Commands/UpgradeUserItemCommand.cs (hunk 1 above)
Comment (verdie-g) [reply src/Application/Items/Commands/UpgradeUserItemCommand.cs:96]: Actually we don't want to unequip it but replace it with the upgraded one

Comment (namidaka) [reply src/Application/Items/Commands/UpgradeUserItemCommand.cs]: image

**Comment by verdie-g** [comment]
Done by https://github.com/verdie-g/crpg/pull/1039.

**Pull request closed without merging**